roles/oidcng/templates/openid-configuration.json.j2

```diff
@@ -0,0 +1,88 @@
+{
+  "issuer": "https://org.openconext.oidc.ng",
```

Comment:
Shouldn't this point to https://oidcng.{{ base_domain }} ?

Reply:
Yep. Changed this.